src/coreclr/System.Private.CoreLib/src/System/Reflection/Emit/ConstructorBuilder.cs

```diff
@@ -23,7 +23,9 @@ public sealed class ConstructorBuilder : ConstructorInfo
 
             m_methodBuilder.GetMethodSignature().InternalGetSignature(out _);
 
+#pragma warning disable IDE0059
```

Comment:
Why use `#pragma` here, when below you used `int _ = ...`?

Reply:
Because https://github.com/dotnet/runtime/pull/63020#pullrequestreview-837612570 so C# compiler complies about variable redefinition

Reply:
I'm not sure I follow. Your link directs to my review comment.

Reply:
It should point to line 24 in that location, there is already `_` variable

Reply:
I believe if you just used

```C#
_ = m_methodBuilder.MetadataToken;
```

It should work.

See https://docs.microsoft.com/en-US/dotnet/csharp/fundamentals/functional/discards

I would also add a comment here to describe why this line is needed (like we do in  src/coreclr/System.Private.CoreLib/src/System/Reflection/Emit/TypeBuilder.cs)

Reply:
Yep, this works